packages/core/src/utils.ts

```diff
@@ -98,26 +98,6 @@ export class Utils {
     return commitData
   }
 
-  /**
-   * Puts a block on IPFS
-   * @param cid the CID of the block to put
-   * @param block bytes array of block to put
-   * @param ipfsApi the IPFS Api instance to use
-   * @param signal AbortSignal
-   */
-  static async putIPFSBlock(
-    cid: CID | string,
```

Comment:
Does this count as a stringly typed fix too 😆

Reply:
LOL